app/assets/javascripts/SourceCodeGlower/SourceCodeLine.js

```diff
@@ -22,26 +24,203 @@ function SourceCodeLine(line_node) {
   this.is_observing      = false;
 }
 
-// Increase a Source Code Line's glow depth
-SourceCodeLine.prototype.glow = function() {
-  // Increase the depth
+// Increase a Source Code Line's glow depth, add listeners
+// Splits nodes at start and end offsets if needed
+SourceCodeLine.prototype.glow = function(annotation_id, start, end,
+  hover_on_function, hover_off_function) {
```

Comment:
Identifier 'hover_on_function' is not in camel case.<br>Identifier 'hover_off_function' is not in camel case.